core/controllers/admin.py

```diff
@@ -848,3 +848,16 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationId(base.BaseHandler):
+    """Handler to show a list of interaction id's for a exploration id."""
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        exploration_id = self.request.get('exploration_id')
+        exploration = exp_fetchers.get_exploration_by_id(exploration_id)
```

Comment:
@BenHenning, Do we want to fetch only the data for the published exploration (or it can be a private exploration as well)?

Reply:
Good question. Since this is the admin page, I'm inclined to say we should support both public & private explorations. Superadmins already have indirect access to private explorations since they can specify admin roles, and we probably will use this for in-progress explorations that we're helping to create. I could see this also being useful for local developers.

Reply:
Thanks for your thoughts, @BenHenning. I agree with Ben that this handler shoul work for both private and public explorations. @aggarwalpulkit596, Can you please make sure that we aren't sending None/null in the intractions list? (Considering a state/card in a private exploration can have interaction ID as None (if the interaction isn't assigned to the state.))

You can do something around:
```
{state.interaction.id for state in exp.states.values() if state.interaction.id is not None}
```

Reply:
updated